src/components/GrampsjsFormSelectType.js

```diff
@@ -85,9 +87,47 @@ class GrampsjsFormSelectType extends GrampsjsTranslateMixin(LitElement) {
               )}
         </mwc-select>
       </p>
+
+      <mwc-button
+        @click="${this.switchTypeInput}"
+      >
+        Enter custom type instead
+      </mwc-button>
+
+      ${this._hasCustomType === false
```

Comment:
Can be `!this._hasCustomType`

Reply:
Changing `${this._hasCustomType === false` to `${!this._hasCustomType` . I agree this is cleaner.